config/settings/test.py

```diff
@@ -38,3 +38,7 @@
 
 # Approvals
 AI_EMPLOYEES_STOCK_DEVELOPER_EMAIL = "colette@ratatouille.com"
+
+# We enable the notifications in the test environment so that can ensure it behaves as expected,
+# then we mock the objects accordingly
```

Comment:
Un peu flou quand on est hors contexte peut-être ? Qu'est-ce que ça sous-entend "*it behaves as expected*" ?

Reply:
je clarifie
```
# We enable the notifications in the test environment so that can ensure the correct API calls are performed,
# but those calls are mocked so that no real data is sent to Pole Emploi
```